src/main/java/org/folio/rest/support/ShelvingOrderUpdate.java

```diff
@@ -0,0 +1,366 @@
+package org.folio.rest.support;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.pgclient.PgConnection;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Transaction;
+import io.vertx.sqlclient.Tuple;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.folio.dbschema.Versioned;
+import org.folio.rest.jaxrs.model.Item;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.persist.PgUtil;
+import org.folio.rest.persist.PostgresClient;
+
+public class ShelvingOrderUpdate {
+  private static final Logger log = LogManager.getLogger();
+
+  public static final String UPDATED_ATTR_NAME = "effectiveShelvingOrder";
+
+  private static final String SQL_SELECT_ITEMS =
+      new StringBuilder()
+          .append("SELECT id AS \"id\", ")
+          .append("jsonb AS \"itemJson\" ")
+          .append(String.format("FROM item WHERE not (jsonb ? '%s') ", UPDATED_ATTR_NAME))
+          .append(" FOR UPDATE ")
+          .append("LIMIT $1").toString();
```

Comment:
```suggestion
  private static final String SQL_SELECT_ITEMS =
          "SELECT id AS \"id\", " +
          "jsonb AS \"itemJson\" " +
          String.format("FROM item WHERE not (jsonb ? '%s') ", UPDATED_ATTR_NAME) +
          " FOR UPDATE " +
          "LIMIT $1";
```
This is more readable and has the same performance. The compiler automatically uses a StringBuilder internally: https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java
The same for the SQL_UPDATE_ITEMS1 and SQL_UPDATE_ITEMS.

Reply:
I see, it will be better to avoid java code in such declarations. As for performance I never had any doubts that usage of plus operation uses builder under the hood. I just used build for sake of readability. I'll fix this, thank you for noticing.